esx_service/vmdk-opsd

```diff
@@ -112,32 +113,65 @@ stop() {
    if [ -n "${PID}" ]; then
       echo "Stopping $OPSD_TAG with PID=${PID}"
       kill -$SIGTERM ${PID}
+
       local COUNT=0
       while kill -0 ${PID} > /dev/null 2>&1; do
          sleep ${WAIT_KILL_SEC};
+         echo "Waiting for operations in flight to complete, please do not interrupt..."
          if [ $((COUNT++)) -gt ${NUM_RETRY} ]; then
-            echo "Sending kill -${SIGKILL} to ${PID} after kill -${SIGTERM} did not stop ${PID}"
-            kill -${SIGKILL} ${PID}
+            echo "Kill -${SIGTERM} did not stop ${PID}"
```

Comment:
Could use a modified version of the original message - Something like "failed to terminate the service after N retries, killing service..."

Reply:
That's not the intent - we are not trying to kill the service after N retries.